e2e/gui/src/test/java/com/epam/pipeline/autotests/LaunchParametersTest.java

```diff
@@ -594,4 +691,12 @@ private void checkFilesystemSpace(ShellAO shell, int diskSize) {
                 .assertNextStringIsVisible(CHECK_SPACE_COMMAND, rootHost)
                 .assertPageAfterCommandContainsStrings(CHECK_SPACE_COMMAND, logMessage(diskSize));
     }
+
+    private String[] expectedLogMessages() {
+        List<String> loglist = new ArrayList<String>();
+        loglist.addAll(asList(command[4].replace("I ", "")
+                .replaceAll("export", "> export").split("\n")));
```

Comment:
'addAll()' call can be replaced with parametrized constructor call

Reply:
Done